source/ObjectViewer/Options.cs

```diff
@@ -10,14 +10,25 @@ internal class Options
     {
         internal static void LoadOptions()
         {
+            string optionsFolder = OpenBveApi.Path.CombineDirectory(Program.FileSystem.SettingsFolder, "1.5.0");
+            if (!System.IO.Directory.Exists(optionsFolder))
+            {
+                System.IO.Directory.CreateDirectory(optionsFolder);
+            }
             CultureInfo Culture = CultureInfo.InvariantCulture;
-            string assemblyFolder = System.IO.Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
-            string configFile = OpenBveApi.Path.CombineFile(OpenBveApi.Path.CombineDirectory(OpenBveApi.Path.CombineDirectory(assemblyFolder, "UserData"), "Settings"), "options_ov.cfg");
+            string configFile = OpenBveApi.Path.CombineFile(Program.FileSystem.SettingsFolder, "1.5.0/options_ov.cfg");
```

Comment:
Marginally better to use the optionsFolder variable declared above in case this changes in the future.